Add documentation for new integration School Holiday (#43876)
home-assistant/home-assistant.github.io: source/_integrations/school_holiday.markdown

---
title: School Holiday
description: Instructions on how to set up the School Holiday integration.
ha_category:
  - Binary sensor
  - Calendar
ha_release: 2026.3
ha_platforms:
  - binary_sensor
  - calendar
ha_iot_class: Cloud Polling
ha_quality_scale: bronze
ha_config_flow: true
ha_codeowners:
  - '@arievanderwende'
ha_domain: school_holiday
ha_integration_type: service
---

The **School Holiday** {% term integration %} helps you automate your home based on the school holidays of the selected country and region.

> The integration provides a binary sensor that indicates whether the current day is a school holiday, which you can use in your {% term automation automations %} to create smart routines that automatically adjust based on school holidays. For example, you can adjust your morning routines or alarm settings to match school days versus school holidays, making your home adapt to your family's schedule.

Reviewer: ```suggestion
The integration indicates whether the current day is a school holiday. You can use this in your {% term automation automations %} to create smart routines that automatically adjust based on school holidays. For example, you can adjust your morning routines or alarm settings to match school days versus school holidays, making your home adapt to your family's schedule.
```
